module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -139,35 +140,6 @@ public function getListTags()
         return $user->getListTags($this->id, $this->user_id);
     }
 
-    /**
-     * Update and save the list object using a request object -- useful for
-     * sharing form processing between multiple actions.
-     *
-     * @param \VuFind\Db\Row\User|bool   $user    Logged-in user (false if none)
-     * @param \Laminas\Stdlib\Parameters $request Request to process
-     *
-     * @return int ID of newly created row
-     * @throws ListPermissionException
-     * @throws MissingFieldException
-     */
-    public function updateFromRequest($user, $request)
```

Comment:
I deleted rather than deprecated this because of its dependency on save (see below).